python/tvm/error.py

```diff
@@ -57,6 +57,11 @@ def __init__(self, msg):
 register_error("KeyError", KeyError)
 
 
+@register_error
+class RPCError(RuntimeError):
+    """Error thrown by the RPC call."""
```

Comment:
can you specify more (I.e. by the transport, by the serializer/deserializer, by the remote server, or by the remote function)?